src/private/teams.ts

```diff
@@ -0,0 +1,49 @@
+import { sendMessageToParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { FrameContexts, SdkError } from '../public';
+
+/**
+ * Namespace to interact with the files specific part of the SDK.
+ *
+ * @private
+ * Hide from docs
+ */
+export namespace teams {
+  export enum ChannelType {
+    Regular = 0,
+    Private = 1,
+    Shared = 2,
+  }
+
+  export interface TeamsChannelInfo {
+    siteUrl: string;
+    objectId: string;
+    folderRelativeUrl: string;
+    displayName: string;
+    channelType: ChannelType;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Get a list of channels belong to a Team
+   * @param teamId a team's objectId
+   */
+  export function getTeamsChannels(
```

Comment:
NIT: Seems like `getTeamChannels` would be more accurate than `getTeamsChannels` since the API only retrieves the channels for one team at a time